chain/network/src/peer_manager/network_state/mod.rs

```diff
@@ -392,6 +448,33 @@ impl NetworkState {
         account_id: &AccountId,
         msg: RoutedMessageBody,
     ) -> bool {
+        if tcp::Tier::T1.is_allowed_routed(&msg) {
```

Comment:
I'd add a comment to the function - to say that the message will be routed via Tier1 or Tier2 depending on the contents.

Reply:
also - the bool might be a little bit confusing - as it returns only the success of the tier2 sending (and the message might have been sent over tier1 succesfully)

Reply:
added comment, made the function return true iff the message is sent over either TIER1 or TIER2.